src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/OracleEngine.java

```diff
@@ -233,7 +239,7 @@ public void connect() throws Exception {
         if (properties.getSchema() != null && properties.getSchema().length() > 0 &&
                         !properties.getUsername().equals(properties.getSchema())) {
             // Set connection schema if one is defined and not the same as the user name
-            Statement stmt = conn.createStatement();
+            final Statement stmt = conn.createStatement();
```

Comment:
how about a try-with-resources, and drop the _stmt.close();_ below?

Reply:
Right, let's clean up code whenever we have the opportunity. Thanks for the remark